### ISSUE TITLE
AgGroup cannot search patterns starting with number

### ISSUE THREAD

**Issue by albfan** [albfan/ag.vim#115]
:AgGroup 0001-foo

Searchs for `-foo`

**Comment by albfan** [comment]
From https://vimhelp.org/map.txt.html (:command-count)

```
-count=N    A count (default N) which is specified either in the line
                    number position, or as an initial argument (like :Next).
```

while 

```
-range=N    A count (default N) which is specified in the line
                    number position (like :split); allows for zero line
                    number.
```

So using `-range` instead of `-count` will fix it

**Comment by albfan** [comment]
fixed on 4afad11
